### DISCUSSION TITLE
Bed Leveling before each print: going back to print progress.

### DISCUSSION THREAD

**Discussion by atamadon** [mriscoc/Ender3V2S1#1351] [General]
I have added the G29 and have tried M500 gcodes to my printer's Machine G-Code profile in my slicer settings to level my bed automatically, however, once leveling, the printer displays the bed mesh and the "Save" and "Continue" button prompts without going back to the print progress screen. 

Is there anyway to make sure the screen goes back to the print progress screen after a bed level and ensure the mesh generated at the start of the print is being used for the print?

Thanks

**Comment by mriscoc** [comment]
Please check the wiki: https://github.com/mriscoc/Ender3V2S1/wiki/Slicer-G-code-Scripts

**Comment by atamadon** [comment]
Thanks! The C108 command closes the mesh viewer.
